Fix aten::to export in ONNX

jamesr66a opened the pull request:
Summary: D10356994 broke ONNX export for casting, this fixes it

Reviewed By: wanchaol

Differential Revision: D10366103

houseroad commented:
@jamesr66a could you please try to enable some of tests disabled in this PR? https://github.com/pytorch/pytorch/pull/12592

houseroad reviewed (commented):
The test is failing, update the expect files?

Commit:
Fix aten::to export in ONNX

Summary: D10356994 broke ONNX export for casting, this fixes it

Reviewed By: wanchaol

Differential Revision: D10366103

Pulled By: wanchaol

fbshipit-source-id: b109036b34d1fffec06b72a2c3e9eee4395c3917
